contract/src/utils.rs

```diff
@@ -20,12 +21,19 @@ use casper_types::{
 };
 
 use crate::{
-    constants::{ARG_TOKEN_HASH, ARG_TOKEN_ID, HOLDER_MODE, OWNED_TOKENS, OWNERSHIP_MODE},
+    constants::{
+        ARG_TOKEN_HASH, ARG_TOKEN_ID, HOLDER_MODE, MAX_PAGE_NUMBER, OWNERSHIP_MODE,
+        PAGE_DICTIONARY_PREFIX,
+    },
     error::NFTCoreError,
     modalities::{NFTHolderMode, NFTIdentifierMode, OwnershipMode, TokenIdentifier},
-    BurnMode, BURNT_TOKENS, BURN_MODE,
+    utils, BurnMode, ALL_TOKEN_OWNERS, BACKFILLED_TOKEN_TRACKER, BURNT_TOKENS, BURN_MODE,
+    FORWARD_TRACKER, IDENTIFIER_MODE, NUMBER_OF_MINTED_TOKENS, OWNED_TOKENS, REVERSE_TRACKER,
+    TOKEN_COUNT_AT_UPGRADE, TOKEN_OWNERS,
 };
 
+pub const PAGE_SIZE: u64 = 10;
```

Comment:
maybe put a comment on this super load bearing magic number

Reply:
Added comment